### ISSUE TITLE
Add an explanation as to the difference between pygerrit and pygerrit2

### ISSUE THREAD

**Issue by oharboe** [dpursehouse/pygerrit2#27]
Why are there two nearly identical pygerrit projects?

This is unusual and it would be nice to have an explanation. There's an note that pygerrit2 is now being maintained, but not why this change was made:

https://github.com/sonyxperiadev/pygerrit/commit/89578cb65abc4cdaa41304663ed9ef206794b681

**Comment by dpursehouse** [comment]
There are a couple of reasons.

1. I developed and released `sonyxperiadev/pygerrit` while I was working at Sony. When I stopped working there in 2016 it wasn't clear whether I would retain any permissions on the project, or if anyone else at Sony was willing to take over maintainership.

2. I wanted to drop support for SSH (and the corresponding dependency on paramiko and friends) and make pygerrit a REST-only client.  However, there are/were still people using the SSH functionality.

So, I decided to fork `sonyxperiadev/pygerrit` to under my own github namespace and release it as a different pypi artifact.  Thus I could remove the SSH support (people who need SSH can continue to use the old one), be guaranteed to still be able to maintain and develop it even if I lose access rights on `sonyxperiadev`, and the original `pygerrit` can still be maintained in parallel if necessary.

As it turns out, I still do have write access on `sonyxperiadev/pygerrit`,  and IIRC @zxiiro was given write access.  But there's no guarantee that those permissions are permanent for either of us.  Someone at Sony could arbitrarily remove us at any time.

**Comment by oharboe** [comment]
thanks!

it might be  a good idea to have something of this on the website/readme
under a history heading.

Den fre. 6. jul. 2018, 03.39 skrev David Pursehouse <
notifications@github.com>:

**Comment by dpursehouse** [comment]
> it might be  a good idea to have something of this on the website/readme under a history heading.

I updated the readme in b3952ec but without going into so much detail.

**Comment by oharboe** [comment]
looks good.

Thanks!



Den fre. 6. jul. 2018, 09.04 skrev David Pursehouse <
notifications@github.com>:
